content/jobs/support-developer.md

```diff
@@ -8,23 +8,38 @@ images = ["https://crunch.io/img/logo-1200x630.png"]
+Crunch is a well-funded, remote-first company with a collaborative and open culture. We are passionate about our product and our mission, and value work/life balance, efficiency, simplicity, and fantastic customer service.
+
+We are hiring a Customer Experience Manager to help us better understand our customers and to improve the product to meet their needs. 
 
 In this position, you will
 
-* Field user questions and reports and triage them appropriately, including identifying clear steps to reproduce issues; contribute to the design of a scalable customer-support system
-* Write public resources (documentation, examples, blog posts) that educate current and potential users on the features of our product and how to most effectively use them
-* Work with product management to refine feature specifications based on your support interactions
-* Collaborate with engineering leads to schedule work on urgent and/or important user-reported issues
-* Draft content and communicate with users, both individually to notify them that requested features/bugs have been addressed, and collectively through in-app announcements, release notices, and mass emails.
+* Provide friendly first line communication for customers 
+* Develop user-training materials and documentation
+* Identify and replicate bugs with the software and help our development team understand and prioritize resolutions
+* Take ownership over the customer support to ensure that none of our users fall through the cracks
+* Analyze customer behavior to Identify and prioritize areas of the app which need improvement
+* Help publicize and communicate new features to customers
+
 
 Professional experience in software development is not a requirement for this position. This is an ideal role for someone who may not be an experienced programmer but who thinks critically and methodically, who has an intuitive sense of good user experience, and who can communicate with and translate between software developers and everyone else. Some experience with writing code will help you succeed in the role, and if you want to become a better programmer, you will have plenty of opportunities to grow.
 
 ## Skills
 
-* Attention to detail
-* Strong analytical and deductive skills, important for isolating the cause of bugs
-* Excellent writing ability: you can express yourself clearly and with appropriate tone and grammar
-* Empathy, both for users and engineers
-* Programming skills, or an interest in developing programming skills, are a plus
-* Strong organizational skills and ability to field multiple concurrent responsibilities without letting any slip through
+Required skills:
+
+* 2 - 5 years experience in market research, survey analysis, or customer support
+* Bachelor’s degree in a quantitative field, or equivalent experience
```

Comment:
I'm not a fan of putting a BA degree here unless we absolutely need it (and in this position we do not). It discourages people who might not have one to apply even with "or equivalent experience". If someone doesn't have the experience we will see that.

Reply:
I'm happy either way, my guess is that we're going to get an overwhelming number of applicants even with the degree requirement